generators/entity-client/templates/react/src/main/webapp/app/entities/_entity.tsx

```diff
@@ -45,11 +54,22 @@ export interface I<%= entityReactName %>Props {
   <%_ } _%>
   match: any;
 }
+<%_ if (searchEngine === 'elasticsearch') { _%>
+
+export interface I<%= entityReactName %>State {
```

Comment:
this could be added to a base interface as it will be needed for all entity, but we can do that while adding sorting etc

Reply:
we will do this in the other PR to avoid conflicts ;)